src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeFile(toFileName, calculateData(readFile(fromFileName)));
+    }
+
+    private StringBuilder readFile(String fileName) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName));) {
+            StringBuilder fromFileNameReader = new StringBuilder();
+            int value = reader.read();
+            while (value != -1) {
+                if ((char) value == ',' || value == 10) {
+                    fromFileNameReader.append(" ");
+                } else if (value == 13) {
```

Comment:
this is hardcode - this is difficult to read
please see the checklist points about magic numbers and constant fields